### PR TITLE
Wildcard option

### PR DESCRIPTION
**Pull request by colincoleman**

### CODE REVIEW

File: outputs.tf [hunk 1]
```diff
@@ -3,5 +3,10 @@
 # ------------------------------------------------------------------------------
 output "arn" {
   description = "The ARN of the certificate that is being validated."
-  value       = "${aws_acm_certificate_validation.main.certificate_arn}"
+  value       = "${aws_acm_certificate_validation.main.*.certificate_arn}"
+}
+
+output "wildcard_arn" {
+  description = "The ARN of the wildcard certificate that is being validated."
+  value       = "${aws_acm_certificate_validation.wildcard.*.certificate_arn}"
```
Comment (mikael-lindstrom) [inline outputs.tf:11]: What happens here if `var.wait_for_validation` is set to false?

Comment (colincoleman) [reply outputs.tf:11]: If wait for validation is set to false then the validation request is made as normal but the terraform script does not wait for confirmation that the validation is complete.  This means that the script cannot return the arn of the created certificate as an output.  It also means that you know the script will not hang and lock everything up.  There is a note added to the readme on why you might wish to set this to false.

**Comment by itsdalmo** [comment]
Ref the discussion here: https://github.com/TeliaSoneraNorge/aws-common-services/issues/231

I think we should be opinionated and not merge this PR.

File: outputs.tf (hunk 1 above)
Comment (mikael-lindstrom) [reply outputs.tf:11]: Sorry for not explaining myself better, if `var.wait_for_validation` is set to `false` this `${aws_acm_certificate_validation.wildcard.*.certificate_arn}`won't exist right? Will the terraform script break?

**Comment by mikael-lindstrom** [comment]
Given the argument @colincoleman mentioned [here](https://github.com/TeliaSoneraNorge/aws-common-services/issues/231#issuecomment-423893377) I think the module should have wildcard support. Even if we are opinionated there could still be reasons for using wildcards right?

File: outputs.tf (hunk 1 above)
Comment (colincoleman) [reply outputs.tf:11]: Using the splat syntax prevents an error - if you set wait_for_validation to false you get:
cert_arn = []

Comment (mikael-lindstrom) [reply outputs.tf:11]: Good stuff, I was unsure how this works 👍

**Comment by mikael-lindstrom** [comment]
This looks good to me, @itsdalmo could you have a look too?

**Review by itsdalmo — APPROVED** [review]
LGTM

**Pull request merged**